dev-test/codegen.ts

```diff
@@ -55,7 +55,11 @@ const config: CodegenConfig = {
     },
     './dev-test/githunt/graphql-declared-modules.d.ts': {
       schema: './dev-test/githunt/schema.json',
-      documents: ['./dev-test/githunt/**/*.graphql', './dev-test-outer-dir/githunt/**/*.graphql'],
+      documents: [
+        './dev-test/githunt/**/*.graphql',
+        './dev-test-outer-dir/githunt/**/*.graphql',
+        '!**/nothing-should-use-this-query.graphql',
```

Comment:
I added the `dev-test-outer-dir/githunt/nothing-should-use-this-query.graphql` because it would otherwise match the pattern `'./dev-test-outer-dir/githunt/**/*.graphql',` (which is only referenced once in this `codegen.ts`, making it a convenient choice), but it's negated by `'!**/nothing-should-use-this-query.graphql',`.

This is one of the crude smoke tests I used to see if the code is working, i.e. verifying that `touch dev-test-outer-dir/githunt/nothing-should-use-this-query.graphql` does **not** trigger a rebuild.